### PR TITLE
Adds note about DaemonSet support

### LINKED ISSUE #5674: Defend for containers not supported as DaemonSet
### Description

We need to add a breaking change in 8.10 RNs, and a note on the Elastic Defend page saying that Elastic Defend DaemonSet deployment in Kubernetes is no longer supported 

### Background & resources

* PRs: 
* Issues/metas: 
* Point of contact: Nick
* Test environments: 

### Which documentation set does this change impact?

ESS and serverless

### ESS release

N/A

### Serverless release

-

### Feature differences

-

### API docs impact

-

### Prerequisites, privileges, feature flags

_No response_

### PR DESCRIPTION
**Pull request by benironside**
Fixes #5674 

Previews: 
[8.10 release notes](https://security-docs_bk_5675.docs-preview.app.elstc.co/guide/en/security/master/release-notes-header-8.10.0.html#breaking-changes-8.10.0)
[Install Elastic Defend](https://elastic-dot-co-docs-production-4skfjfmbc-elastic-dev.vercel.app/current/serverless/security/install-edr)

### CODE REVIEW

**Comment by elasticdocs** [comment]
:rocket: Built _elastic-dot-co-docs-preview-docs_ successfully!

* https://elastic-dot-co-docs-production-1b54rfxh7-elastic-dev.vercel.app

* Built with commit 76799fa2b5d3ebbceca78efdcf6bd95654508a63

> Issues? Visit #next-docs in Slack

**Comment by elasticdocs** [comment]
:rocket: Built _elastic-dot-co-docs-preview-docs_ successfully!

* https://elastic-dot-co-docs-production-4skfjfmbc-elastic-dev.vercel.app

* Built with commit 76799fa2b5d3ebbceca78efdcf6bd95654508a63

> Issues? Visit #next-docs in Slack

File: docs/getting-started/install-endpoint.asciidoc [hunk 1]
```diff
@@ -19,6 +19,8 @@ Like other Elastic integrations, {elastic-defend} is integrated into the {agent}
 
 If you're using macOS, some versions may require you to grant Full Disk Access to different kernels, system extensions, or files. Refer to <<elastic-endpoint-deploy-reqs, requirements for {elastic-endpoint}>> if you're installing the {elastic-endpoint} or <<endgame-sensor-full-disk-access, requirements for the Endgame sensor>> for more information.
 
+NOTE: Elastic Defend is not supported for deployment as a DaemonSet in Kubernetes.
```
Comment (joepeeples) [inline docs/getting-started/install-endpoint.asciidoc:22]:
```suggestion
NOTE: {elastic-defend} is not supported for deployment as a DaemonSet in Kubernetes.
```

File: docs/release-notes/8.10.asciidoc [hunk 2]
```diff
@@ -86,7 +86,7 @@ There are no user-facing changes in 8.10.2.
 [discrete]
 [[breaking-changes-8.10.0]]
 ==== Breaking changes
-There are no breaking changes in 8.10.0.
+* Deploying Elastic Defend as a DaemonSet in Kubernetes is no longer supported.
```
Comment (joepeeples) [inline docs/release-notes/8.10.asciidoc:89]:
```suggestion
* Deploying {elastic-defend} as a DaemonSet in Kubernetes is no longer supported.
```

File: docs/serverless/edr-install-config/install-elastic-defend.mdx [hunk 3]
```diff
@@ -29,6 +29,10 @@ Like other Elastic integrations, ((elastic-defend)) is integrated into the ((age
 
 If you're using macOS, some versions may require you to grant Full Disk Access to different kernels, system extensions, or files. Refer to <DocLink slug="/serverless/security/elastic-endpoint-deploy-reqs">requirements for ((elastic-endpoint))</DocLink> if you're installing the ((elastic-endpoint)) or <DocLink slug="/serverless/security/endgame-sensor-full-disk-access">requirements for the Endgame sensor</DocLink> for more information.
 
+<DocCallOut title="Note">
+Elastic Defend is not supported for deployment as a DaemonSet in Kubernetes.
```
Comment (joepeeples) [inline docs/serverless/edr-install-config/install-elastic-defend.mdx:33]:
```suggestion
((elastic-defend)) is not supported for deployment as a DaemonSet in Kubernetes.
```

**Review by joepeeples — COMMENTED** [review]
Added some quick fixes — can approve once you commit

File: docs/release-notes/8.10.asciidoc (hunk 2 above)
Comment (joepeeples) [reply docs/release-notes/8.10.asciidoc:89]: On second thought -- was removal of support this backported to 8.10?

Comment (joepeeples) [inline docs/release-notes/8.10.asciidoc:89]: Just noticed this -- so was this removal of support backported to 8.10? Typically I'd expect the previous versions to support something, and then in the current version we stop supporting it.

But if we're removing support even from previous versions, then we'd need to update ALL the release notes files for all the versions that we no longer support (so make the same change to 8.11.asciidoc, 8.12.asciidoc, etc.).

Comment (benironside) [reply docs/release-notes/8.10.asciidoc]: The removal of support started with 8.10, but wasn't documented at the time. That's why I think it makes sense to just add it to the 8.10 RNs. Nothing about this feature changed in the 8.11 etc. versions

Comment (joepeeples) [reply docs/release-notes/8.10.asciidoc]: Gotcha, makes sense. Thanks!

**Review by joepeeples — APPROVED** [review]
:shipit:

**Review by nick-alayil — CHANGES_REQUESTED** [review]
This looks good to me but to be technically 💯  accurate. 

elastic defend cannot be deployed as part of elastic agent DaemonSet in Kubernetes

File: docs/release-notes/8.10.asciidoc
Comment (nick-alayil) [inline docs/release-notes/8.10.asciidoc:1]: Similarly to be technically accurate:

elastic defend no longer supports deployment within elastic agent DaemonSet in Kubernetes environments

**Review by nick-alayil — DISMISSED** [review]

**Review by nastasha-solomon — APPROVED** [review]

**Pull request merged**